src/components/Pollquote/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+
+interface PullQuoteProps {
+  //     /** Text for the poll quote */
```

Comment:
poll -> pull, cause we're [_pulling_](https://en.wikipedia.org/wiki/Pull_quote) quotes from the larger article.

Reply:
will do